tests/integration/test_vulnerability_detector/test_providers/data/test_cases/test_no_os.yaml

```diff
@@ -0,0 +1,59 @@
+-
+  name: 'Amazon Linux'
```

Comment:
```suggestion
-  name: 'Amazon Linux'
```

Reply:
Repeat this for the rest

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2547/commits/90bed077f7ebf72dd21e5d15b1af1bc674ba3704